src/machines/timer_state_machine.rs

```diff
@@ -362,30 +349,22 @@ mod test {
         assert!(act
             .unwrap_err()
             .to_string()
-            .contains("Timer fired event did not have expected timer id realid"));
+            .contains("Timer fired event did not have expected timer id 0"));
         wfm.shutdown().await.unwrap();
     }
 
     #[fixture]
     fn cancellation_setup() -> ManagedWFFunc {
         let func = WorkflowFunction::new(|mut cmd_sink: WfContext| async move {
-            let cancel_timer_fut = cmd_sink.timer(StartTimer {
-                timer_id: "cancel_timer".to_string(),
-                start_to_fire_timeout: Some(Duration::from_secs(500).into()),
-            });
-            cmd_sink
-                .timer(StartTimer {
-                    timer_id: "wait_timer".to_string(),
-                    start_to_fire_timeout: Some(Duration::from_secs(5).into()),
-                })
-                .await;
+            let cancel_timer_fut = cmd_sink.timer(Duration::from_secs(500));
+            cmd_sink.timer(Duration::from_secs(5)).await;
             // Cancel the first timer after having waited on the second
-            cmd_sink.cancel_timer("cancel_timer");
+            cmd_sink.cancel(&cancel_timer_fut);
             cancel_timer_fut.await;
             Ok(().into())
         });
 
-        let t = canned_histories::cancel_timer("wait_timer", "cancel_timer");
+        let t = canned_histories::cancel_timer("1", "0");
```

Comment:
I'm not necessarily saying we *need* to change this, but going from readable names to just numbers harms the readability of these tests a decent amount. It would be nice if we could still attach readable IDs to things with sequences somehow, but I understand it's possibly more work than it's worth.